dallinger/heroku/worker.py

```diff
@@ -2,21 +2,26 @@
 
 from future.builtins import map
 import os
-
 import redis
-from rq import (
-    Worker,
-    Queue,
-    Connection
-)
 
-listen = ['high', 'default', 'low']
 
+listen = ['high', 'default', 'low']
 redis_url = os.getenv('REDIS_URL', 'redis://localhost:6379')
-
 conn = redis.from_url(redis_url)
 
-if __name__ == '__main__':
+if __name__ == '__main__':  # pragma: nocover
```

Comment:
Not totally clear what this move inside the `if __name__ ...` was about, but I'm sure it was a good idea. Short code comment?

Reply:
These imports are inside the __main__ block to make sure that we only import from rq_gevent_worker (which has the side effect of applying gevent monkey patches) in the worker process. This way other processes can import the redis connection without that side effect.

I added the above as a comment here.

Really we should probably move the redis connection to a different module.